src/TypedValue/TypedValueFormatter.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Wikibase\DataModel\Services\TypedValue;
+
+use DataValues\DataValue;
```

Comment:
need to fix composer.json to require this? :/
Didn't spot this one when looking through the list earlier